solucoes/desafio_01/miriamretka/test/desafio01Test.exs

```diff
@@ -0,0 +1,11 @@
+ExUnit.start
+
+defmodule Desafio01Test do
+  use ExUnit.Case
+
+  # Teste de demonstração, ainda não há uma maneira padrão de fazer
+  # checagem de tipos em Elixir.
+  test "teste_demo" do
+    assert 1 == 1
+  end
+end
```

Comment:
Esse teste case pode remover, ou acertar pra testar sua função 😛

Reply:
Coloquei para validar se o String é de texto mesmo. Se a saída da função for um bitstring (<<exemplo>>), ela retorna `false` e falha no teste.